scripts/analytics/benchmarks.py

```diff
@@ -139,6 +140,93 @@ def get_log_counts(users):
 
 
 def main():
+
+    number_users = User.find().count()
+
+    projects = Node.find(
```

Comment:
It is better to put these query into method instead of putting them all in main for future editing. like https://github.com/CenterForOpenScience/osf.io/pull/2920/files#diff-c43831b47bf025b14bcba1b381bd8e16R24